models/solution/parsers/stop_parser.rb

```diff
@@ -36,7 +37,7 @@ def self.parse(service, options)
         pickup_shipment_id: service.type == :pickup ? (service.original_id || service.id) : nil,
         delivery_shipment_id: service.type == :delivery ? (service.original_id || service.id) : nil,
         type: service.type,
-        alternative: options[:index],
+        alternative: alternative_index,
```

Comment:
Isn't it better to return `alternative: nil` which will be eliminated by solution parsing so that the user has a way to distinguish between services with alternative and without one?